src/main/java/sirius/biz/storage/layer2/BasicBlobStorageSpace.java

```diff
@@ -1410,6 +1411,16 @@ protected String lookupPhysicalKey(String blobKey, @Nonnull String variantName,
         }
     }
 
+    /**
+     * Ensures that the desired variant exists.
+     *
+     * @param blob        the blob for which the variant should exist
+     * @param variantName the name of the desired variant
+     */
+    public void ensureVariantExists(B blob, String variantName) {
+        findOrCreateVariant(blob, variantName, false);
+    }
+
```

Comment:
as we no longer need this, shouldn't we kill it?

Reply:
yep we could kill `ensureVariantExists`